helm/cluster-apps-operator/values.yaml

```diff
@@ -6,7 +6,7 @@ chartOperator:
   catalog: default
   version: 2.20.0
 
-base: example.gigantic.io
+baseDomain: ""
```

Comment:
I assume this also needs to be changed in the `config` repo?

Reply:
I see is configured [here](https://github.com/giantswarm/config/blob/main/installations/hubble/config.yaml.patch#L1) though not sure we need to maintain old value to no break existing apps. Thomas you know?

Reply:
This is already done here https://github.com/giantswarm/config/pull/808